core/flyout_base.js

```diff
@@ -610,6 +610,7 @@ Blockly.Flyout.prototype.stepScrollAnimation = function() {
  * @private
  */
 Blockly.Flyout.prototype.clearOldBlocks_ = function() {
+  Blockly.Events.disable();
```

Comment:
If you don't need events when clearing the blocks, you also don't need them when creating the blocks.  In this PR or a followup, wrap block creation (in `show`) in a disable/enable pair.

Reply:
Actually I'm going to leave this out for now, I have the feeling some of the monitor related stuff may depend on events from the flyout.